test_util/Makefile

```
@@ -0,0 +1,55 @@
+SHELL := /bin/bash -euo pipefail
+
+export AWS_REGION := us-west-2
+export TF_VAR_custom_dcos_download_path_win := https://downloads.dcos.io/dcos/testing/master/windows/dcos_generate_config_win.sh
+export TF_VAR_variant := open
+
+.DEFAULT_GOAL := test
+
+.PHONY: get-terraform
+get-terraform:
+	rm -rf ./terraform_0.11.14_linux_amd64.zip
+	wget https://releases.hashicorp.com/terraform/0.11.14/terraform_0.11.14_linux_amd64.zip
+	unzip ./terraform*.zip
+	chmod +x terraform
```

We should not install Terraform if it is already present. 
```suggestion
    if ! command -v terraform >/dev/null 2>&1; then
        rm -rf ./terraform_*.zip
	    wget https://releases.hashicorp.com/terraform/0.11.14/terraform_0.11.14_linux_amd64.zip
	    unzip ./terraform*.zip
	    chmod +x terraform
    fi
```
Alternatively we could run the whole thing in a Docker container. Check out Tim's work on Marathon's [packager](https://github.com/mesosphere/marathon/blob/master/tools/packager/Makefile#L5).

---

Hi @jeschkies  - I could easily add some checks to use the existing terraform, however, I think, it might have some disadvantages.
* Terraform is very strict with versioning and 1.11 is not compatible with 1.12 ; So we will have to check with the version.
* We don't want to have variations with minor versions of terraform too.

Using the downloaded, and pinned version of terraform sounds better to me.

----

On using dockerfile, I will check that. For now, since we are running the tests from VM (TC agents) using agents directly seems useful to get both the debug output and access to logs directly.